src/lib/pickles/step_main.ml

```diff
@@ -345,14 +346,15 @@ let step_main :
         and prevs =
           exists (Prev_typ.f prev_proof_typs) ~request:(fun () ->
               Req.Proof_with_datas )
-        and unfinalized_proofs =
+        and unfinalized_proofs_unextended =
```

Comment:
This name change is wrong now: we're getting the *unextended* `unfinalized_proofs` here.